entry_tasks/89_sequence_type_check/main.py

```diff
@@ -0,0 +1,49 @@
+def parse_value(string_value):
+    if string_value.isdigit():
+        return int(string_value)
+    elif string_value.strip().replace('.', '').isdigit():
+        return float(string_value)
+    elif string_value.strip().lower() == "false":
+        return False
+    elif string_value.strip().upper() == "False":
```

Comment:
I think:
```py
elif string_value.strip().lower() == "false":
        return False
```
already covers this (so it's duplicated condition which can be removed, right?)